iris/static/js/lecturer_session.js

```diff
@@ -1,38 +1,42 @@
 const button_start = document.getElementById("button_start");
 const button_stop = document.getElementById("button_stop");
-const feedbackmsg = document.getElementById("lecturer_message")
+const feedbackmsg = document.getElementById("lecturer_message");
 let socket = io();
 socket.emit('join', {'course_id': courseID});
 
-if(sessionActive == "True"){
+function disableStart(){
     button_start.disabled = true;
     button_stop.disabled = false;
-}else{
+    feedbackmsg.innerHTML = "Session active";
+}
+
+function disableStop(){
```

Comment:
This function can probably be generalized. Right now we have two functions doing the exact same thing. Really, this is just a toggle. Using something like `button_start.disabled = !button_start.disabled` should work, and doing that to both lines would give a simple toggle function, instead of two "hardcoded" ones.